hackathon_site/dashboard/frontend/src/assets/abstracts/_mixins.scss

```diff
@@ -0,0 +1,41 @@
+@mixin flexPosition ($jusCon: space-between, $alIte: center) {
+	display: flex;
+	justify-content: $jusCon;
+	align-items: $alIte;
+}
+
+@mixin responsive($breakpoint) {
```

Comment:
This file is inconsistent about whether or not there are spaces before parenthesis - stick with space or no space everywhere